README.md

```diff
@@ -1 +1,50 @@
-# portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Porfolio.
+
+An online portfolio that represents skills, projects, About the person, and contact form  
+
+
+## Built With
+
+- HTML ,CSS
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+
+
+### Install
+git clone git@github.com:Zainab-Alzubair/portfolio.git
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/Zainab-Alzubair)
+- Twitter: [@twitterhandle](@zainabm34401029)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/zainab-al-zubair-bb6777168/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
[OPTIONAL] Kindly add a link to the issues page of your project. Currently this opens an empty link

Reply:
Will do so, thank you for your feedback @Nyame-Wolf